src/handlers/foo.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function(req, res) {
+  // get name from URL parameter
+  // http://localhost:3000/foo?name=foo
+  let kibun = req.query['kibun'] || 'so-so';
+  res.render('foo', {message2: 'You are '+kibun+ ' now.'} );
+
+  // let name = req.query['name'] || 'nanasi';
+  // res.render('foo', {title: 'greeting', message: 'Hello ' + name + '-san.'});
```

Comment:
This is not essential for this p-r's purpose, separating codes.
However, it causes an error to call `render` twice 😭